Scripts/DCS-BIOS/test/mocking_compile/socket.lua

```diff
@@ -0,0 +1,6 @@
+module("socket", package.seeall)
```

Comment:
Is this mock still needed? I thought I removed the global socket calls when I did the io refactor?